src/components/BorrowerProfile/LendCta.vue

```diff
@@ -798,6 +815,29 @@ export default {
 	},
 	mounted() {
 		this.createWrapperObserver();
+		const erlCampaign = this.$route.query.utm_campaign;
+		const topCookie = 'erl-five-notes-top';
+		const baseCookie = 'erl-five-notes-base';
+		const sessionTimestamp = new Date();
+		sessionTimestamp.setHours(sessionTimestamp.getHours() + 24);
+
+		const topUpLenders = 'topup-vb-balance-MPV1';
+		const baseLenders = 'base-vb_balance_MPV1';
+		this.activeCookie = isErlCookieActive(this.cookieStore);
+
+		if (this.enableFiveDollarsNotes) {
+			if (this.activeCookie !== '') {
+				this.selectedOption = this.getDropdownERL(this.userBalance, this.unreservedAmount);
+			} else if (erlCampaign) {
+				if (erlCampaign === topUpLenders) {
```

Comment:
We can't guarantee the case of URLs, so compare these strings with `toUpperCase`